src/ontology/foodon-edit.owl

```diff
@@ -18198,6 +18198,142 @@ Dioscorea japonica is used for food.</obo:IAO_0000115>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003911 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003911">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_00003083"/>
+        <obo:IAO_0000115 xml:lang="en">A meat of a calf that is raised on a fortified milk formula diet plus solid feed.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119 rdf:datatype="http://www.w3.org/2001/XMLSchema#anyURI">https://en.wikipedia.org/wiki/Veal</obo:IAO_0000119>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2021-08-07T20:53:21Z</dc:date>
+        <oboInOwl:hasExactSynonym xml:lang="en">formula-fed veal</oboInOwl:hasExactSynonym>
+        <oboInOwl:hasExactSynonym xml:lang="en">milk fed veal</oboInOwl:hasExactSynonym>
+        <oboInOwl:hasExactSynonym xml:lang="en">milk-fed veal</oboInOwl:hasExactSynonym>
+        <oboInOwl:hasExactSynonym xml:lang="en">special fed veal</oboInOwl:hasExactSynonym>
+        <oboInOwl:hasExactSynonym xml:lang="en">special-fed veal</oboInOwl:hasExactSynonym>
+        <oboInOwl:hasExactSynonym xml:lang="en">white veal</oboInOwl:hasExactSynonym>
+        <rdfs:label xml:lang="en">formula fed veal</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003912 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003912">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_03450006"/>
+        <obo:IAO_0000115 xml:lang="en">A Japanese cooking technique in which food is broiled or grilled with a glaze of soy sauce, mirin, and sugar.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119 rdf:datatype="http://www.w3.org/2001/XMLSchema#anyURI">https://en.wikipedia.org/wiki/Teriyaki</obo:IAO_0000119>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2021-08-07T21:03:32Z</dc:date>
+        <rdfs:label xml:lang="en">teriyaki</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://purl.obolibrary.org/obo/FOODON_00003913 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/FOODON_00003913">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/FOODON_03309952"/>
+        <obo:IAO_0000115 xml:lang="en">An ingredient or a combination of ingredients containing extra minerals, vitamins, and other compounds, is added to the cattle feed in order to meet specific needs.</obo:IAO_0000115>
+        <obo:IAO_0000117 rdf:resource="https://orcid.org/0000-0001-5275-8866"/>
+        <obo:IAO_0000119 xml:lang="en">https://www.bentoli.com/cattle-supplements-benefits-value/</obo:IAO_0000119>
+        <dc:date rdf:datatype="http://www.w3.org/2001/XMLSchema#dateTime">2021-08-07T21:13:22Z</dc:date>
+        <rdfs:label xml:lang="en">cattle supplement</rdfs:label>
```

Comment:
relable "cattle feed supplement", and exact synonym cattle supplement if necessary?